common-dependencies-V1/pom.xml

```diff
@@ -150,7 +150,7 @@
                 *:hbase-server,*:protobuf-java
               </include>
               <exclude>
-                *:*log4j*,*:xml-apis,*:xercesImpl,*:hadoop*,*:hbase*,*:zookeeper*
+                *:*log4j*,*:xml-apis,*:xercesImpl,*:hadoop*,*:hbase*,*:zookeeper*,com.fasterxml.jackson.core:jackson-databind
```

Comment:
Do you what dependency is introducing "com.fasterxml.jackson.core:jackson-databind" ?
are there any other jackson dependencies that are now being introduced?

looking at the artifact pentaho-hadoop-shims-common-dependencies-V1-10.2.0.0-217.jar it contains:

- META-INF/
- META-INF/MANIFEST.MF
- META-INF/maven/
- META-INF/maven/org.pentaho.hadoop.shims/
- META-INF/maven/org.pentaho.hadoop.shims/pentaho-hadoop-shims-common-dependencies-V1/
- META-INF/maven/org.pentaho.hadoop.shims/pentaho-hadoop-shims-common-dependencies-V1/pom.properties
- META-INF/maven/org.pentaho.hadoop.shims/pentaho-hadoop-shims-common-dependencies-V1/pom.xml
- htrace-core-3.1.0-incubating.jar
- httpclient-4.5.14.jar
- mxparser-1.2.2.jar
- pentaho-hadoop-shims-common-mapreduce-10.2.0.0-217.jar
- protobuf-java-2.5.0.jar
- xmlpull-1.1.3.1.jar
- xstream-1.4.20.jar

Reply:
Dont remember right now but there were a lot. Mostly the hadoop libraries.

Reply:
looking at s/data-integration/system/karaf/system/com/pentaho/hadoop/shims/pentaho-hadoop-shims-ee-apache-driver/10.3.0.0-45/pentaho-hadoop-shims-ee-apache-driver-10.3.0.0-45.jar


the following files were present:

- jackson-annotations-2.14.2.jar
jackson-core-asl-1.9.13.jar
jackson-databind-2.14.2.jar
jackson-jaxrs-1.9.13.jar
jackson-jaxrs-base-2.14.2.jar
jackson-jaxrs-json-provider-2.14.2.jar
jackson-mapper-asl-1.9.13.jar
jackson-module-jaxb-annotations-2.14.2.jar

I'd check a more recent 10.3 build and be parity with the jars and of course update the version.

Reply:
I re-read the scope, you are suppose to remove.